src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -621,7 +621,13 @@ def _validate_vm_vmss_accelerated_networking(cli_ctx, namespace):
                       'Standard_L16s_v2', 'Standard_L32s_v2', 'Standard_L64s_v2', 'Standard_L96s_v2', 'SQLGL',
```

Comment:
Does the service not provide a useful error message in this case?  At what point do we remove the client-side validation and leave it up to the service? It seems we are artificially limiting the service capability with this.

Reply:
ETA for the API support will be by the end of this year, and once it is available we will remove all of those.

Reply:
You mean like an endpoint to query this information?  What happens if you just provide incompatible values?  How long does the service take to error out and what message does it provide?

Reply:
yes, the same endpoint used by `az vm list-skus`
Blindly turning on will fail the "vm create" after a minute or so, which is the case for our default size of `Standard_DS1_v2`. So retrying with the flag off will have perf issue.